config-templates/aws/aws-ebs-csi-driver/0.9.14/metadata.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "0.9.14",
+  "enabled": "true",
+  "description": "AWS EBS CSI Driver version 0.9.14",
+  "tags": [
+    {
+      "name": "provider",
+      "value": "aws"
+    },
+    {
+      "name": "offering-name",
+      "value": "aws-ebs-csi-driver"
+    },
+    {
+      "name": "provider-url",
+      "value": "https://console.aws.amazon.com/support/home#/"
+    },
+    {
+      "name": "support-msg",
+      "value": "For support contact AWS"
+    },
+    {
+      "name": "provider-logo",
+      "value": "/aws-ebs-logo"
+    },
+    {
+      "name": "supported-envs",
+      "value": "AWS"
+    },
+    {
+      "name": "supported-k8s-versions",
+      "value": "1.17,1.18"
+    },
+    {
+      "name": "volume-type",
+      "value": "block"
+    },
+    {
+      "name": "volume-encryption-at-rest",
+      "value": "true"
+    },
+    {
+      "name": "volume-encryption-in-transit",
+      "value": "true"
+    },
+    {
+      "name": "volume-snapshot",
+      "value": "true"
+    },
+    {
+      "name": "volume-availability",
+      "value": "zonal"
+    },
+    {
+      "name": "volume-expansion",
+      "value": "true"
+    },
+    {
+      "name": "storageclass",
+      "value": "gold"
```

Comment:
Update it to "value": "gold,silver,bronze".
Also update for existing version `0.8.2`